.circleci/create-virtualenv.sh

```diff
@@ -47,3 +47,7 @@ export PIP_FIND_LINKS="file://${WHEELHOUSE_PATH}"
 # above, it may still not be able to get us a compatible version unless we
 # explicitly ask for one.
 "${PIP}" install --upgrade setuptools==44.0.0 wheel
+
+# Just about every user of this image wants to use tox from the bootstrap
+# virtualenv so go ahead and install it now.
+"${PIP}" install "tox~=3.0"
```

Comment:
We used to install tox into the virtualenv in `populate-wheelhouse.sh` but now `populate-wheelhouse.sh` really only populates the wheelhouse, it doesn't also install stuff into the virtualenv.